src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmVMSqlServerAutoPatchingConfig.md

```diff
@@ -1,5 +1,5 @@
 ---
-external help file: Microsoft.Azure.Commands.Compute.dll-Help.xml
+external help file: Microsoft.WindowsAzure.Commands.ServiceManagement.dll-Help.xml
```

Comment:
Why do you replace Microsoft.Azure.Commands.Compute.dll-Help.xml  with Microsoft.WindowsAzure.Commands.ServiceManagement.dll-Help.xml ?